### ISSUE TITLE
The functionality add attachment to parts allows access to local files.

### ISSUE THREAD

**Issue by alestorm980** [partkeepr/PartKeepr#1229]
[comment]: # (Hi and thanks for an issue. In order to help you best, please fill out the following information.)

# Bug description

In PartKeepr before **v1.4.0**, the functionality to load attachments using a URL when creating a part, allows the use of the **file://** URI scheme, allowing local files to be read.

## Steps to reproduce

1. Go to 'Add Part'.
2. Click on 'Attachments'.
2. Click on 'Add'.
3. Fill the 'URL' field with "file:///etc/passwd".
4. Click on the uploaded file in order to see the content.

## Expected behavior

The application should not allow access to local files.

## Observed behavior

Local paths can be used to read files on the system.

## Screenshots and files

![path_file](https://user-images.githubusercontent.com/17485471/148068290-56dffec4-08a9-419a-b5ce-251460835cde.png)

![passwd_content](https://user-images.githubusercontent.com/17485471/148068319-47f2bd8b-4590-448b-ab5b-cd1723911b54.png)

## System Information

* PartKeepr Version: v1.4.0 and v0.1.9
* Operating System: Linux
* Web Server: Apache
* PHP Version: 7.4
* Database and version: Mysql
* Reproducible on the demo system: Yes.

**Comment by Gasman2014** [comment]
This is how most users add locally stored / downloaded data sheets / images etc. Does this allow access to resources that the user does not have permissions for?

**Comment by alestorm980** [comment]
Yes, it is possible to read files within the server to which the user running the application has access, this includes source code, system configuration files, ssh keys, etc.

**Comment by alestorm980** [comment]
I attach the link to the advisory https://fluidattacks.com/advisories/hendrix/
